### ISSUE TITLE
Component - OdsTextToggleButtonsRow

### ISSUE THREAD

**Issue by paulinea** [Orange-OpenSource/ods-android#530]
Add a text toggle buttons row component :
![TextToggleRow](https://github.com/Orange-OpenSource/ods-android/assets/17410558/95e92b29-f950-4b73-a07a-06ee61151b86)

**Comment by afrahB** [comment]
in light mode the contrast for the selected element is not sufficient @mccart77 
it's ok for the dark mode

**Comment by mccart77** [comment]
Contrast colour ratio for the selected tab does not meet the accessibility criteria, as Highlighted by Accessibility team.

The issue has been discussed in the Mobile Specification Meeting - the following was decided:

**For light mode**
For the selected text tab, the text colour will be updated to be black (#000000) for all interactive states. The selected background colour will stay as it is (orange). 

**For dark mode**
For the selected text tab, the text colour will be updated to be white (#FFFFFF) for all interactive states. The selected background colour will stay as it is (orange).

**Comment by paulinea** [comment]
> Contrast colour ratio for the selected tab does not meet the accessibility criteria, as Highlighted by Accessibility team.
> 
> The issue has been discussed in the Mobile Specification Meeting - the following was decided:
> 
> **For light mode** For the selected text tab, the text colour will be updated to be black (#000000) for all interactive states. The selected background colour will stay as it is (orange).
> 
> **For dark mode** For the selected text tab, the text colour will be updated to be white (#FFFFFF) for all interactive states. The selected background colour will stay as it is (orange).

@afrahB @mccart77 This is ok on our side, you will be able to test it in the today release.
